mypy_boto3_builder/stubs_static/types-aioboto3/dynamodb/table.pyi

```diff
@@ -34,7 +34,7 @@ class BatchWriter:
     async def __aenter__(self: _R) -> _R: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc_value: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc_value: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aioboto3/blob/main/aioboto3-stubs/dynamodb/table.pyi